masonry/src/widget/portal.rs

```diff
@@ -20,6 +20,7 @@ use crate::{
 
 // TODO - refactor - see https://github.com/linebender/xilem/issues/366
 // TODO - rename "Portal" to "ScrollPortal"?
+// TODO - Document which cases need request_layout, request_compose and request_render
```

Comment:
Is there a reason this isn't being done in this PR?

Reply:
Because it's more work than the rest of the PR combined.